### PR TITLE
MODINV-1066: Update mapping for "sourceRecordIdentifier" property in …

### PR DESCRIPTION
**Pull request by PBobylev**
…"instance_ingress" event

### CODE REVIEW

**Review by AndreiBordak — APPROVED** [review]

**Review by pkjacob — APPROVED** [review]

**Comment by pkjacob** [comment]
Hi @PBobylev, We should not merge this to "master" branch right? If we merge "beta" will break (because mod-linked-data will be sending instance ID in sourceRecordId field and mod-inventory is expecting instance ID in "additionalProperties -> instanceID" field.

**Review by askhat-abishev — APPROVED** [review]

**Review by KaterynaSenchenko — APPROVED** [review]

**Pull request merged**
